src/components/Explore/SpeciesView.js

```diff
@@ -67,11 +73,42 @@ const SpeciesView = ( {
     {
       ...queryParams,
       fields: {
-        taxon: Taxon.TAXON_FIELDS
+        taxon: Taxon.SCORE_IMAGE_FIELDS
       }
     }
   );
 
+  const taxonIds = data.map( r => r.taxon.id );
+
+  const { data: seenByCurrentUser } = useAuthenticatedQuery(
+    ["fetchSpeciesCounts", taxonIds],
+    ( ) => fetchSpeciesCounts( {
```

Comment:
You need to use `optsWithAuth` to actually authenticate this query and get the correct preferred_common_name values in the response.

Reply:
I didn't think we needed `optsWithAuth` since we're only returning `taxonIds` from this query, and taxon ids don't need translation or any current user preferences...

Reply:
True, but in that case this doesn't need to be authenticated, right?